TestRandom.java

```diff
@@ -1,6 +1,21 @@
 // Tests the distribution generated by Java's Math.random() function.
 public  class  TestRandom {
 	public static void main(String[]  args) {
-	    // Replace this comment with your code
+		int lower = 0;
+		int bigger = 0;
+		int length = Integer.parseInt(args[0]);
+		for (int i = 0; i < length; i++) {
+			if (Math.random() > 0.5) {
+				bigger++;
+				} else {
+				lower++;
+				}
```

Comment:
Indentation!
This closing curly bracket should be in the same vertical line as the "i" in the `if`.